README.md

```diff
@@ -334,6 +344,7 @@ Companies that have allowed us to speak about our work can be found here. Many m
 
 | Product | Review Date | Level of Effort | Announcement |
 | --- | --- | --- | --- |
+| [Compound Chain](https://compound.finance/) May 2021 | 6 person-weeks | 
```

Comment:
This sounds like a smart contract

Reply:
This was a blockchain review, but it is confusing because the previous implementation was a smart contract. They actually include a single Ethereum smart contract inside (to be able to lock value from the Ethereum network onto their new blockchain system), which we may have audited before, separately, but it was out of scope for this audit. They are doing something new and it's a new blockchain which is what we audited.